generator/internal/descriptor_utils.cc

```diff
@@ -439,6 +439,11 @@ ParameterCommentSubstitution substitutions[] = {
     {"`filter='strategy=TrafficPercentStrategy'`",
      "``filter='strategy=TrafficPercentStrategy'``"},
 
+    {"conversionWorkspace/123/mappingRules/"
```

Comment:
I can see that both of these strings are in the doxygen comments for the data migration client, but I don't know why there is a mapping from "text" -> "`text`"

In the client, it looks like we are using both in line 1963 and line 1968.

Reply:
Because `@foo` (or more precisely `@c7cfa2a8c7cfa2a8c7cfa2a8c7cfa2a8`) is treated as a Doxygen command unless is enclosed in backticks.  Doxygen breaks because that is not a valid command.